.gitignore

```diff
@@ -2,5 +2,5 @@ bin/
 obj/
 docs/
 latex/
-.coverage/
+.coverage**
```

Comment:
Wieso?

Reply:
Ich hab bei mir den report in nem anderem Unterordner (`.converage-report`). Fand ich einfach übersichtlicher, weil im `.coverage` Ordner ja das ganze XML Zeug abgeladen wird

Reply:
Wieso machst gibst du den output nicht als `"-targetdir:.coverage/coverage-report/"` beim reportgenerator an?

Reply:
Achso, ja, stimmt... Das könnte ich natürlich machen